stock_inventory_discrepancy/models/res_config_settings.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    inventory_discrepancy_enable = fields.Boolean(
+        string="Inventory Discrepancy Control",
+        help="Enable to show line discrepancies in inventory and block validation "
+        "if the discrepancy exceeds the threshold.",
```

Comment:
```suggestion
        help="Block validation of the inventory adjustment if discrepancy exceeds the threshold.",
```
Since the setting doesn't affect the UI.